xmltodict.py

```diff
@@ -175,17 +175,17 @@ def push_data(self, item, key, data):
             key, data = result
         if item is None:
             item = self.dict_constructor()
-        try:
-            value = item[key]
-            if isinstance(value, list):
-                value.append(data)
-            else:
-                item[key] = [value, data]
-        except KeyError:
+        not_present = object() # guaranteed to not be in <item>
```

Comment:
Can't you simply do `item.get(key, None)`?

Reply:
It would change the semantics, losing any keys with `None` as value from the input dict. I don't know if that matters for xmltodict specifically, but it seems safer to use a unique sentinel to conserve even `None` values.